addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -380,17 +364,12 @@ public void packetReceived(byte[] packet) {
                 } else if (message instanceof RFXComTransmitterMessage) {
                     RFXComTransmitterMessage resp = (RFXComTransmitterMessage) message;
 
-                    byte seqNbr = getSeqNumber();
-                    if (resp.seqNbr == seqNbr) {
-                        logger.debug("Transmitter response received: {}", message.toString());
-                        setResponseMessage(resp);
-                        synchronized (notifierObject) {
-                            notifierObject.notify();
-                        }
-                    } else {
-                        logger.warn("Sequence number '{}' does not match, expecting number '{}'", resp.seqNbr, seqNbr);
-                    }
+                    logger.debug("Transmitter response received: {}", resp.toString());
 
+                    if (resp.response != Response.NAK)
+                        transmitQueue.sendNext();
+                    else
+                        transmitQueue.resend();
```

Comment:
The only time it will give us a NAK is if the transmitter fails to lock on frequency. All other errors will just lead to, um, undefined behaviour. A failure to lock _may_ be transient or _may_ be recoverable by replugging (turning it off and on again)  but... all these protocols are unACKed on the RF side and expected to be lossy. So, no. On balance you're right and there's no point trying. Just move along and try the next.